### PR TITLE
Added word count as required by the dataLayer implementation

### PR DESCRIPTION
**Pull request by sulfo**
The dataLayer require the word count number. Here's the implementation of it.

### CODE REVIEW

File: src/Actions/Backend/EstimatedReadingTime.php [hunk 1]
```diff
@@ -7,12 +7,35 @@
...
-    public function addEstimatedReadingTime($postId) {
+    public function countNumberOfWords($postId)
+    {
+        $compositeAdapter = new CompositeAdapter(get_post($postId));
+        $totalWordCount = 0;
+
+        foreach ($compositeAdapter->getContents() as $item) {
+            switch ($item->getType()) {
+                case 'infobox':
+                case 'text_item':
```
Comment (MFlor) [inline src/Actions/Backend/EstimatedReadingTime.php]: What about description fields on galleries and hotspot images, the lead paragraph, quote and paragraph list widget?

Comment (sulfo) [reply src/Actions/Backend/EstimatedReadingTime.php]: Not for now. infobox and text_item is the only two things we are interested in.

File: style.css [hunk 2]
```diff
@@ -4,7 +4,7 @@ Theme URI: https://github.com/benjaminmedia/willow-base-theme
 Author: Bonnier Publications
 Author URI: https://github.com/benjaminmedia
 Description: Willow Base theme for WordPress - A full WordPress REST API - no frontend at all.
-Version: 1.17.1
+Version: 1.17.2
```
Comment (MFlor) [inline style.css]: You are adding functionality, so according to semantic versioning, you need to update the "minor"-number. The version should be 1.18.0

File: src/Models/Contracts/Composites/CompositeContract.php [hunk 3]
```diff
@@ -71,4 +71,6 @@ public function getEstimatedReadingTime(): ?int;
     public function getAssociatedComposites(): ?Collection;
 
     public function getAudio(): ?AudioContract;
+
+    public function getWordCount(): ?int;
```
Comment (MFlor) [inline src/Models/Contracts/Composites/CompositeContract.php]: Why make the returned int nullable, if you default to `0` everywhere?
Either default to `null` or change the return type to be just `int`.

Comment (sulfo) [reply src/Models/Contracts/Composites/CompositeContract.php]: Good point.

File: src/Adapters/Wp/Composites/CompositeAdapter.php [hunk 4]
```diff
@@ -350,4 +350,9 @@ public function getAudio(): ?AudioContract
         }
         return null;
     }
+
+    public function getWordCount(): ?int
+    {
+        return intval(get_post_meta($this->getId(), 'word_count', true)) ?: null;
```
Comment (MFlor) [inline src/Adapters/Wp/Composites/CompositeAdapter.php]: If there 0 words on a given composite, the method will return null - is that intended behaviour?

**Pull request merged**